src/order/filter/ennreal.lean

```diff
@@ -0,0 +1,66 @@
+/-
+Copyright (c) 2021 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+
+import data.real.ennreal
+import order.filter.countable_Inter
+import order.liminf_limsup
+
+/-!
+# Order properties of extended non-negative reals
+
+This file compiles order-related results about `ennreal` (see data/real/ennreal.lean).
```

Comment:
```suggestion
This file compiles filter-related results about `ennreal` (see data/real/ennreal.lean).
```
Could you also comment that it could make sense for more general typeclasses, but since the only relevant example is `ennreal` this is not worth the trouble?